clients.md

```diff
@@ -44,7 +44,8 @@ Client libraries make it easier to access the API from your programming language
 
 ## Python
 
-* [python-api-client](http://github.com/meetup/python-api-client) encapsulates all the logic needed to make queries to the Meetup API in Python.
+* [python-api-client](http://github.com/meetup/python-api-client) encapsulates all the logic needed to make queries to the Meetup API in Python. (discontinued)
+* [python-api](https://github.com/pferate/meetup-api) Python API for Meetup.
```

Comment:
looks like this repo is out of maintenance?

Reply:
Hm. I've been [idly trying to get in touch](https://twitter.com/PFerate/status/942960570638012416) to see if @pferate is willing to graciously add more maintainers to help bring it back to life, but "no dice" so far

Regardless, should be marked as "(maintenance)" before merging this? Thanks! :)

Reply:
👍